js/node.js

```diff
@@ -0,0 +1,56 @@
+const emailDiv = document.querySelector('#email');
+const pwdDiv = document.querySelector('#password');
+const emailInput = document.querySelector('.email-input');
+const pwdInput = document.querySelector('.password-input');
+
+const pwdInput2 = document.querySelector('.password-input-check');
+const pwdDiv2 = document.querySelector('#password-check');
+
+function inputAddNode(type, message, errorCase) {
```

Comment:
```suggestion
function inputAddNode({type, message, errorCase}) {
```

파라미터를 받을때, 전 개인적으로 3개가 넘어가면 객체형태로 받는편이에요
파라미터가 많아졌을때, 순서에 의존적으로 파라미터를 넘기다보면 헷갈리거든요
넘길때 inputAddNode({type:'example', message:'example', errorCase: 'example} 요렇게 더 명시적으로 넘길수 있어요! 객체니까 순서를 바꾸어도 되고용!